iocore/net/P_NetVConnection.h

```diff
@@ -81,3 +85,22 @@ NetVConnection::get_local_port()
 {
   return ats_ip_port_host_order(this->get_local_addr());
 }
+
+TS_INLINE sockaddr const *
```

Comment:
Should be able to use `inline` instead of `TS_INLINE` now that we're C++ eleventy.